test/sanity/validate-modules/skip.txt

```diff
@@ -1,2 +1,2 @@
 lib/ansible/modules/utilities/logic/async_status.py
-lib/ansible/modules/utilities/helper/_accelerate.py
+lib/ansible/modules/utilities/helper/_accelerate.py
```

Comment:
@asand3r looks like you are overwriting this file.

Reply:
@trishnaguha you know, I've misunderstand the error at morning and deleted skip.txt:
"test/sanity/validate-modules/ignore.txt:538:1: A102 Remove since "lib/ansible/modules/network/ios/ios_logging.py" passes "E322" test"
So now, I've placed it back and all test are fine, as I can see. Looks like that needs nothing to fix. Right?

Reply:
@asand3r if you check the main ansible devel branch the file already has the content that you just added https://github.com/ansible/ansible/blob/devel/test/sanity/validate-modules/skip.txt.
So even if you have mistakenly removed that and re-added git shouldn't show the change, it is showing the change because you might have overwritten.

I see it is there since your last 3 commits. The solution would be to remove the last 3 commits and force push. Make sure you have your branch rebased with the main devel branch.

Reply:
@trishnaguha OK, I've removed my last three commits and make force push. So now, I've got same error:
```code
2018-01-18 12:03:24 ERROR: Found 1 validate-modules issue(s) which need to be resolved:
2018-01-18 12:03:24 ERROR: test/sanity/validate-modules/ignore.txt:538:1: A102 Remove since "lib/ansible/modules/network/ios/ios_logging.py" passes "E322" test (75%)
2018-01-18 12:03:24 Sanity check using yamllint
2018-01-18 12:03:24 Run command: python3.6 test/sanity/yamllint/yamllinter.py test/integration/targets/ios_logging/tests/cli/basic.yaml
2018-01-18 12:03:24 Run command: python3.6 test/sanity/yamllint/yamllinter.py lib/ansible/modules/network/ios/ios_logging.py
2018-01-18 12:03:24 ERROR: The 1 sanity test(s) listed below (out of 41) failed. See error output above for details.
```
I've checked ansible devel branch and see that ignore.txt contains exactly same strigs as my. Sorry, I can't see where I making mistake. =)